mod/file/thumbnail.php

```diff
@@ -6,7 +6,12 @@
  */
 
 // Get engine
-require_once __DIR__ . '/../../vendor/autoload.php';
+$initialRoot = dirname(dirname(__DIR__));
+$backupRoot = dirname(dirname(dirname($initialRoot)));
+
+if (!include_once "$initialRoot/vendor/autoload.php") {
```

Comment:
Can you just require this file? https://github.com/Elgg/Elgg/blob/2.x/autoloader.php